### ISSUE TITLE
Link rendered as button is displayed as normal text(no color differentiation) in high contrast mode from office ui fabric .

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#14482]
Link rendered as button displayed as normal text in high contrast mode.

### Environment Information

OS: Win10
Browser: Edge

### Describe the issue:
Link rendered as button is displayed as normal text(no color differentiation) in high contrast mode  .

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/v-ajrai/pen/GRZKwOE?editable=true%3Dhttps%3A%2F%2Fdeveloper.microsoft.com%2Fen-us%2Ffluentui
![copy_id_ext1](https://user-images.githubusercontent.com/62233982/89978130-69644d00-dc8a-11ea-824a-2abd1570b6be.png)
![copy_id_ext_2](https://user-images.githubusercontent.com/62233982/89978138-6d906a80-dc8a-11ea-9624-1dd0fca738a3.png)

#### Actual behavior:
Link rendered as button is displayed as normal text(no color differentiation) in high contrast mode .

#### Expected behavior:
Link rendered as button  should be differentiable from normal text(no color differentiation) in high contrast mode .

**Comment by paulgildea** [comment]
@smhigley FYI

@xugao Do you know if this has been addressed with the recent high contrast fixes?

**Comment by khmakoto** [comment]
I can confirm that this is happening. However, I'm not really sure on how to fix this since the `Link` high contrast style is not something that we can access. We could definitely `Highlight` the text in another way or with another color but I'm not sure if that's what we'd want.

@betrue-final-final any ideas?

**Comment by betrue-final-final** [comment]
Without a visible border, this control is effectively invisible. I'm not sure how to fix this, other than assigning "Hotlight" to it, but does this ever render with borders?
